### PR TITLE
Create 1.20.0 release notes

### PR DESCRIPTION
**Pull request by keirthana**
Draft of 1.20 release notes - we need to add streaming improvements.

### CODE REVIEW

**Comment by keirthana** [comment]
@adglkh Could you suggest the streaming improvements that need to be added? I see lot of items are in "Rejected", so don't want to get in the wrong items.

**Comment by keirthana** [comment]
@morphis Do we still hold back on calling out vulkan support? I see there is still some work to do in the next cycle.

File: release-notes/1.20.0.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* Improvements to session streaming statistics
+* Ability to delete instances in bulk
+* Ability to change the Anbox Cloud configuration
+* Simplified app version selection for new sessions
+* Improvements to general design and layout
+* Warning messages to improve user experience
+
+### Other
+
+* Android security updates for November 2023 (see Android Security Bulletin - November 2023 for more information).
```
Comment (ajanon) [inline release-notes/1.20.0.md:31]:
```suggestion
* Android security updates for November 2023 (see [Android Security Bulletin - November 2023](https://source.android.com/docs/security/bulletin/2023-11-01) for more information).
```

**Review by ajanon — CHANGES_REQUESTED** [review]

File: release-notes/1.20.0.md (hunk 1 above)
Comment (keirthana) [reply release-notes/1.20.0.md]: Thanks, added now.

File: release-notes/1.20.0.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.20.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* Introduced support for creating virtual machine instances for use cases that require a stronger isolation. With the 1.20 version, you can use both containers and virtual machines for your applications.<!--AC-1459-->
+* Implemented lazy image download to download images only when necessary. AMS will download the image from the image server at first use. <!--AC-1158-->
```
Comment (morphis) [inline release-notes/1.20.0.md:12]: Lets call out alpha support for Vulkan graphics

`* Alpha quality support for Vulkan on NVIDIA GPUs is now available. Enabling requires newer NVIDIA drivers (>= 545) and is kept behind a feature flag. Bugs and crashes are known and expected at this point. Stable production support is expected for the 1.21 release of Anbox Cloud. If you are interested in early testing, please reach out.`

Comment (morphis) [reply release-notes/1.20.0.md:12]: `* The required NVIDIA driver version has been updated from 525 to 535.`

**Review by morphis — CHANGES_REQUESTED** [review]
I've added a few missing parts but looks like we miss quite a few other things as well (see https://warthogs.atlassian.net/issues/?jql=fixVersion%20%3D%201.20.0%20AND%20labels%20%3D%20release-notes).

File: release-notes/1.20.0.md (hunk 2 above)
Comment (keirthana) [reply release-notes/1.20.0.md]: Thanks! I have gotten everything in now except AC-1923, I will add that after a discussion

**Comment by adglkh** [comment]
> @adglkh Could you suggest the streaming improvements that need to be added? I see lot of Jira items are in "Rejected", so don't want to get in the wrong items.

A: Hmm, when I checked the [streaming improvements](https://warthogs.atlassian.net/browse/AC-1885) for the JIRA ticket, I saw one JIRA was rejected,
    - https://warthogs.atlassian.net/browse/AC-1885 
   Other than that, three JIRA tickets were triaged, 
    - https://warthogs.atlassian.net/browse/AC-1587
    - https://warthogs.atlassian.net/browse/AC-1980
    - https://warthogs.atlassian.net/browse/AC-2020
    The last one is injected at the last minute for this sprint, so for the above three we'd continue with that in the next sprint.

File: release-notes/1.20.0.md (hunk 2 above)
Comment (keirthana) [reply release-notes/1.20.0.md]: Done for AC-1923 as well

**Comment by keirthana** [comment]
> > @adglkh Could you suggest the streaming improvements that need to be added? I see lot of Jira items are in "Rejected", so don't want to get in the wrong items.
> 
> A: Hmm, when I checked the [streaming improvements](https://warthogs.atlassian.net/browse/AC-1885) for the JIRA ticket, I saw one JIRA was rejected, - https://warthogs.atlassian.net/browse/AC-1885 Other than that, three JIRA tickets were triaged, - https://warthogs.atlassian.net/browse/AC-1587 - https://warthogs.atlassian.net/browse/AC-1980 - https://warthogs.atlassian.net/browse/AC-2020 The last one is injected at the last minute for this sprint, so for the above three we'd continue with that in the next sprint.

Thanks, I checked all of these and they are for later than 1.20. I think we are good with adding the alpha support for Vulkan for NVIDIA and the NVIDIA driver version updates.

File: release-notes/1.20.0.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+* AMS downloads the image from the image server only at first use instead of downloading all available images. <!--AC-1158-->
+* The watchdog can be enabled without specifying a boot package in the application manifest. <!--AC-2057-->
+* The Anbox Cloud images no longer start `android.hardware.media.c2` services automatically.<!--AC-2033-->
+* The included Mesa driver is tracking the latest 23.2.1 version and the `virglrenderer` is also tracking the latest upstream.<!--AC-1933-->
+* Some improvements to the swagger response for the AMS HTTP API are included. <!--AC-1923-->
+
+### Streaming improvements
+
+* The required NVIDIA driver version has been updated from the 525 series to 535 series.<!--AC-1924-->
+* Alpha quality support for Vulkan on NVIDIA GPUs is now available through a feature flag. To use this feature, you require newer NVIDIA drivers (versions 545 or later). Since the feature is at Alpha level, bugs and crashes can occur. If you are interested in early testing, contact us through the [users forum on discourse](https://discourse.ubuntu.com/c/anbox-cloud/users/148). Stable product support for this feature is planned for the 1.21 release.
```
Comment (adglkh) [inline release-notes/1.20.0.md:22]: Experimental features for video degradation to support dynamic adaptive streaming in constrained-bandwidth situations in the [Anbox WebRTC streamer](https://anbox-cloud.io/docs/ref/webrtc-streamer). Below is an example of the streamer JSON file with this experimental feature enabled for 720p screen format:
```
{
  "video": {
    "experimental": {
      "degradation": {
        "level": 0.1,
        "texture_filter_mode": "linear"
      }
    },
    "bitrate_limits": [
      {
        "num_pixels": 921600,
        "min_kbps": 512,
        "max_kbps": 5000
      }
    ]
  }
}
```

Comment (adglkh) [reply release-notes/1.20.0.md:22]: @keirthana as discussed, let's leave this out as it's still experimental. And if customers are interested in this feature, come and talk to us.

File: release-notes/1.20.0.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+* The Mesa build in `/opt/mesa` is removed. <!--AC-1928-->
+* The Grafana and Prometheus tools for monitoring are removed. The Anbox Cloud roadmap has plans of providing a monitoring solution using the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack). <!--AC-1925-->
+
+## Bug fixes
+The following bugs are fixes in the Anbox Cloud 1.20 version:
+
+* Permission errors when Android tries to access cgroup. <!--AC-1748-->
+* Missing support for VPU slots for `amc` commands. <!--AC-2004-->
+
```
Comment (adglkh) [inline release-notes/1.20.0.md:59]: * Anbox IME remaining popped up when streaming from a tablet

File: release-notes/1.20.0.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* Anbox Cloud has introduced support for creating virtual machine instances. This is especially helpful for use cases that require a stronger isolation. You can use both containers and virtual machines for your applications.
+
+  For easy understanding, we have introduced a change in terminology as well. The *Anbox Cloud containers* or *LXD containers* are now called *Instances*. The term *Instance* in the [Anbox Cloud documentation](https://anbox-cloud.io/docs) denotes a container or a virtual machine that hosts an application or an image. <!--AC-1459 and AC-2010 and AC-1916-->
```
Comment (adglkh) [inline release-notes/1.20.0.md:13]: I feel like it's worth mentioning the creation of GPU based VM instance is unsupported in 1.20.0 release.

File: release-notes/1.20.0.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+* The Anbox Management Service (AMS) snaps use [core22](https://snapcraft.io/core22).<!--AC-1853-->
+* AMS downloads the image from the image server only at first use instead of downloading all available images. <!--AC-1158-->
+* The watchdog can be enabled without specifying a boot package in the application manifest. <!--AC-2057-->
+* The Anbox Cloud images no longer start `android.hardware.media.c2` services automatically.<!--AC-2033-->
+* The included Mesa driver is tracking the latest 23.2.1 version and the `virglrenderer` is also tracking the latest upstream.<!--AC-1933-->
+* Some improvements to the swagger response for the AMS HTTP API are included. <!--AC-1923-->
```
Comment (adglkh) [inline release-notes/1.20.0.md:19]: AOSP screen timeout is disabled by default

File: release-notes/1.20.0.md (hunk 5 above)
Comment (keirthana) [reply release-notes/1.20.0.md:13]: Done, thanks!

File: release-notes/1.20.0.md (hunk 4 above)
Comment (keirthana) [reply release-notes/1.20.0.md]: The first three are added as bug fixes and the last one is added as a core stack improvement (as discussed)

File: release-notes/1.20.0.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+  Also, in the Anbox Cloud documentation, the use of the term *Instance type* to indicate a set of resources is deprecated and will be completely removed in the future releases. Instead, the term *Resource preset* is used to indicate a set of resources available for an instance.
+
+## Removed functionality
+
+* The Mesa build in `/opt/mesa` is removed. <!--AC-1928-->
+* The Grafana and Prometheus tools for monitoring are removed. The Anbox Cloud roadmap has plans of providing a monitoring solution using the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack). <!--AC-1925-->
+
+## Bug fixes
+The following bugs are fixes in the Anbox Cloud 1.20 version:
```
Comment (ajanon) [inline release-notes/1.20.0.md:56]:
```suggestion
The following bugs are fixed in the Anbox Cloud 1.20 version:
```

**Review by ajanon — CHANGES_REQUESTED** [review]
One nitpick

File: release-notes/1.20.0.md (hunk 7 above)
Comment (keirthana) [reply release-notes/1.20.0.md]: Thanks for catching this typo.

**Review by ajanon — APPROVED** [review]
LGTM

**Review by adglkh — APPROVED** [review]
LGTM

**Review by morphis — APPROVED** [review]
LGTM

**Review by jat-canonical — APPROVED** [review]
LGTM

**Pull request merged**